pantheon-decoupled.php

```diff
@@ -45,13 +45,53 @@ function pantheon_decoupled_graphql_smart_object_cache() {
 }
 
 function pantheon_decoupled_settings_init() { 
-    add_options_page( 'Pantheon Front-End Sites', 'Pantheon Front-End Sites', 'manage_options', 'pantheon-front-end-sites', 'pantheon_decoupled_options_page' );
+    add_options_page( 'Pantheon Front-End Sites', 'Pantheon Front-End Sites', 'manage_options', 'pantheon-front-end-sites', 'pantheon_decoupled_settings_page' );
+
+	add_settings_section(
```

Comment:
I'm seeing some layout funkyness on the FES settings page now.

<img width="1560" alt="Screenshot at Aug 10 16-05-14" src="https://github.com/pantheon-systems/wp-pantheon-decoupled/assets/889478/33a26621-0912-4f58-a918-ece9f181a6bd">

It looks like all of this content is being treated as part of a table now and an empty header row is being inserted. My guess is that this is related to how `add_settings_section` / `add_settings_field` is being used here, but I'm not familiar enough with this API to know why yet.

Reply:
This is because `do_settings_sections` renders the sections as a table. I switched to using `do_settings_fields` which ditches that 👍 

https://developer.wordpress.org/reference/functions/do_settings_sections/#more-information